### PR TITLE
11.0 mig stock picking mass action

### PR DESCRIPTION
**Pull request by mpanarin**
migration of the module to v11.0

### CODE REVIEW

File: stock_picking_mass_action/tests/test_mass_action.py [hunk 1]
```diff
@@ -49,38 +49,50 @@ def setUpClass(cls):
         })
 
     def test_mass_action(self):
-        self.assertEqual(self.picking.state, 'draft')
+        self.assertEqual(self.picking.state, 'confirmed')
```
Comment (rousseldenis) [inline stock_picking_mass_action/tests/test_mass_action.py:52]: Has picking flow changed (default is 'confirmed' ?) ?

File: stock_picking_mass_action/wizard/mass_action.py [hunk 2]
```diff
@@ -23,51 +22,72 @@ def _default_force_availability(self):
     def _default_transfer(self):
         return self.env.context.get('transfer', False)
 
+    def _default_picking_ids(self):
+        self.write({
```
Comment (rousseldenis) [inline stock_picking_mass_action/wizard/mass_action.py]: Quite weird even in a wizard. Maybe update() is more correct

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comments

File: stock_picking_mass_action/__manifest__.py [hunk 3]
```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
-# Copyright 2014 Camptocamp SA - Guewen Baconnier
+# Copyright 2018 Camptocamp SA
```
Comment (TDu) [inline stock_picking_mass_action/__manifest__.py]: Shouldn't the copyright keep the original date like this 2014-2018 ?
And it is the same in other files

File: stock_picking_mass_action/tests/test_mass_action.py (hunk 1 above)
Comment (mpanarin) [reply stock_picking_mass_action/tests/test_mass_action.py:52]: pickings are now autoconfirmed on creation
https://github.com/odoo/odoo/blob/11.0/addons/stock/models/stock_picking.py#L508

**Review by yvaucher — APPROVED** [review]

**Pull request merged**